### ISSUE TITLE
Option to restart Ceph services

### ISSUE THREAD

**Issue by ommoreno** [ceph/cbt#102]
When changing Ceph parameters, some of them require Ceph services, like OSD,  to be restarted. CBT can only recreate a brand new cluster or use existing cluster. It would be nice to have something like use_existing, but it would restart all Ceph services and check health before starting the test(s).

**Comment by bengland2** [comment]
Typically CBT creates only brand new clusters for test purposes.  IMHO We don't want to re-invent ceph-ansible, for example.    CBT clusters may not be persistent across reboots, and it's not a requirement.  If you want that kind of cluster, I'd suggest creating the cluster with ceph-ansible and then test it with CBT using "use_existing: True".

I think if you change Ceph parameters in the file specified via **conf_file** in YAML, then it will propagate that ceph.conf to all hosts _before_ the services are started, so that's the way to change ceph parameters when using CBT to build a cluster, right?

**Comment by ommoreno** [comment]
That makes sense. I agree with using ceph-ansible/ceph-deploy to create persistent Ceph clusters.

Yep, building a cluster with CBT pointing to a modified conf_file is usually the way I do it. It's a bit of a pain though to have to recreate pool/vols and fill, which may take a long time.
